Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenWindowControllerIOS.mm

```diff
@@ -1601,6 +1597,7 @@ - (void)_performSpatialFullScreenTransition:(BOOL)enter completionHandler:(Compl
             [ornament _setDepthDisplacement:originalDepth + kIncomingWindowZOffset];
 
         [UIView animateWithDuration:kWindowTranslationDuration delay:0 options:UIViewAnimationOptionCurveEaseInOut animations:^{
+            ornament.viewController.view.window.alpha = enter ? 0.0 : 1.0;
```

Comment:
This should be done in the fade animation block.

Reply:
Also the incoming/outgoing fade is different.

Reply:
fixed!